### PR TITLE
Add Duck Player prime modal

### PR DESCRIPTION
**Pull request by CrisBarreiro**
Task/Issue URL: https://app.asana.com/0/0/1207812780384049/f 

### Steps to test this PR

_Feature 1_
- [x] Open Duck Player
- [x] Click Info button
- [x] Check prime modal is correctly shown in both landscape and portrait

### UI changes
Screenshot_20240802_180603

Screenshot_20240802_180616

### CODE REVIEW

**Comment by CrisBarreiro** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4824?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4824** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4824?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4780** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4780?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4752** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4752?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4748** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4748?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4663** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4663?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @CrisBarreiro and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by marcosholgado** [comment]
One problem that I'm seeing on my S10 Android 12. When the screen is in landscape the X to close the popup is over/below the bottom bar and the X is unclickable.

Screenshot 2024-08-07 at 16 24 23

File: app/src/main/java/com/duckduckgo/app/browser/duckplayer/DuckPlayerJSHelper.kt [hunk 1]
```diff
@@ -168,6 +169,9 @@ class DuckPlayerJSHelper @Inject constructor(
             "openSettings" -> {
                 return OpenDuckPlayerSettings
             }
+            "openInfo" -> {
```
Comment (marcosholgado) [inline app/src/main/java/com/duckduckgo/app/browser/duckplayer/DuckPlayerJSHelper.kt:172]: Any tests for this?

File: content-scope-scripts/content-scope-scripts-impl/src/main/java/com/duckduckgo/contentscopescripts/impl/messaging/ContentScopeScriptsJsMessaging.kt [hunk 2]
```diff
@@ -133,6 +133,7 @@ class ContentScopeScriptsJsMessaging @Inject constructor(
             "sendDuckPlayerPixel",
             "setUserValues",
             "openDuckPlayer",
+            "openInfo",
```
Comment (marcosholgado) [inline content-scope-scripts/content-scope-scripts-impl/src/main/java/com/duckduckgo/contentscopescripts/impl/messaging/ContentScopeScriptsJsMessaging.kt:136]: Should we test this type of message too?

**Review by marcosholgado — APPROVED** [review]
LGTM, left just two nits.

File: app/src/main/java/com/duckduckgo/app/browser/duckplayer/DuckPlayerJSHelper.kt (hunk 1 above)
Comment (CrisBarreiro) [reply app/src/main/java/com/duckduckgo/app/browser/duckplayer/DuckPlayerJSHelper.kt:172]: Tested in next PR, https://github.com/duckduckgo/Android/pull/4872

**Pull request merged**